src/gcodeExport.h

```diff
@@ -218,23 +217,29 @@ class GCodeExport : public NoCopy
 
     /*!
      * Get the gcode file header (e.g. ";FLAVOR:UltiGCode\n")
-     * 
+     *
      * \param extruder_is_used For each extruder whether it is used in the print
      * \param print_time The total print time in seconds of the whole gcode (if known)
      * \param filament_used The total mm^3 filament used for each extruder or a vector of the wrong size of unknown
      * \param mat_ids The material GUIDs for each material.
      * \return The string representing the file header
      */
-    std::string getFileHeader(const std::vector<bool>& extruder_is_used, const Duration* print_time = nullptr, const std::vector<double>& filament_used = std::vector<double>(), const std::vector<std::string>& mat_ids = std::vector<std::string>());
+    std::string getFileHeader(const std::vector<bool>& extruder_is_used,
+                              const Duration* print_time = nullptr,
+                              const std::vector<double>& filament_used = std::vector<double>(),
+                              const std::vector<std::string>& mat_ids = std::vector<std::string>());
+
+    void setSliceUUID(const std::string& slice_uuid);
 
     void setLayerNr(unsigned int layer_nr);
 
     void setOutputStream(std::ostream* stream);
 
-    bool getExtruderIsUsed(const int extruder_nr) const; //!< return whether the extruder has been used throughout printing all meshgroup up till now
+    bool getExtruderIsUsed(
+      const int extruder_nr) const; //!< return whether the extruder has been used throughout printing all meshgroup up till now
```

Comment:
Why did you break it up here, rather than at the comment?

I'd suggest to write the comment as an actual documentation block instead so that Doxygen can pick up on it.

Reply:
Done we the current clang-format in this branch. Will change this manually for now to 4 spaces so I don't get a merge conflict if the fix failing tests is merged